cfgov/jinja2/v1/_includes/organisms/filterable-list.html

```diff
@@ -169,7 +165,11 @@
                                         'upload': post.preview_image,
                                         'alt': post.preview_image.alt if post.preview_image.alt else '',
```

Comment:
Indentation fix, while we're here.

```suggestion
                                            'upload': post.preview_image,
                                            'alt': post.preview_image.alt if post.preview_image.alt else '',
```